internal/driver/driver.go

```diff
@@ -208,6 +211,25 @@ func applyCommandOverrides(cmd string, outputFormat int, cfg config) config {
 	return cfg
 }
 
+// withoutEmptyStrings filters a slice to only non-empty strings
+func withoutEmptyStrings(in []string) (out []string) {
```

Comment:
Could you order functions top-down rather than bottom-up? This seems more common in Go as it allows to focus on the high level code logic first when reading the code top down.

Reply:
Reordered it below